specs/rails/db_migrate.yaml

```diff
@@ -0,0 +1,11 @@
+---
+name: Apply database migrations
```

Comment:
Same feedback here as above, let's include Rails here to make the name a little more specific:
"Apply Rails Database Migrations"